### ISSUE TITLE
Make doc site more readable

### ISSUE THREAD

**Issue by tylergannon** [rethinkdb/rethinkdb#6958]
Just a simple request, for the benefit of persons whose eyesight is not very acute, I'd like to ask you to increase the contrast on your web site, between the base font and the background.

I normally don't do this but I spent a minute on your site asking myself "why is this thing so hard to read" before noting that your font is both slim and colored somewhat lightly, and your background isn't helping the matter.

If it helps to set a standard, Github's web site also features fonts so I tend to keep `Dark Mode` enabled so that I can read it better, but your web site doesn't offer that.

thanks!

**Comment by srh** [comment]
I totally agree, I think the text should generally be solid black text.  I'll take a look at it, starting with the docs.

(If you want my sense of documentation aesthetic, see my [hosted copy of the Horizon documentation](http://samuelhughes.com/rethinkdb/horizon-docs/docs/index.html), but maybe we won't go *that* far...

**Comment by srh** [comment]
I'm trying to remember what this commit actually affected: https://github.com/rethinkdb/docs/commit/3c560cd00bdefc9db627fd6ffbdc8bbbd19e1522
